lib/plugins/aws/invokeLocal/index.js

```diff
@@ -336,7 +343,14 @@ class AwsInvokeLocal {
       context = customContext;
     }
 
-    return lambda(event, context, callback);
+    const maybeThennable = lambda(event, context, callback);
+    if (maybeThennable !== undefined && typeof maybeThennable.then === 'function') {
```

Comment:
we should use lodash here `if (!_.isUndefined(maybeThennable) && _.isFunction(maybeThennable.then)) {`